ml/Host.h

```diff
@@ -41,6 +41,8 @@ class RrdHost {
 
     std::map<RRDDIM *, Dimension *> DimensionsMap;
     std::map<Dimension *, std::mutex> LocksMap;
+    //The window size of anomaly bit counting for anomaly percentage
+    double SaveAnomalyPercentageEvery = 15 * 60;
```

Comment:
I think I misled you with a previous comment I made about not making this configurable. I still think that it should be under `ml/Config.{h,cc}` (because this is were every other configurable option lives) just use a constant value instead of setting the value with `config_get_*()`. Having this here looks a little bit out of place.

Reply:
:) changed.